<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="G-0439" name="orion3_Navigation.AdmissionsTab" tests="8" failures="2" timestamp="18 Jun 2015 09:08:45 GMT" time="97.670" errors="0">
  <testcase name="verifySearchRepPage" time="26.405" classname="orion3_Navigation.AdmissionsTab"/>
  <testcase name="verifySearchLeadPage" time="8.430" classname="orion3_Navigation.AdmissionsTab"/>
  <testcase name="verifyReassignLeadsDelegationPage" time="0.002" classname="orion3_Navigation.AdmissionsTab">
    <failure type="org.testng.SkipException" message="Charlie Login Credentials window is appearing DefectID: - null">
      <![CDATA[org.testng.SkipException: Charlie Login Credentials window is appearing DefectID: - null
	at orion3_Listeners.DisableKnownTestFailures.beforeInvocation(DisableKnownTestFailures.java:47)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker$InvokeBeforeInvocationWithoutContextStrategy.callMethod(InvokedMethodListenerInvoker.java:84)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker.invokeListener(InvokedMethodListenerInvoker.java:62)
	at org.testng.internal.Invoker.runInvokedMethodListeners(Invoker.java:619)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:687)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- verifyReassignLeadsDelegationPage -->
  <testcase name="verifyAdmissionsManagerPage" time="38.158" classname="orion3_Navigation.AdmissionsTab"/>
  <testcase name="verifySearchLeadAllNewPage" time="3.904" classname="orion3_Navigation.AdmissionsTab"/>
  <testcase name="verifyOpsUpdateArchivePage" time="7.900" classname="orion3_Navigation.AdmissionsTab"/>
  <testcase name="verifyReassignLeadsPage" time="0.002" classname="orion3_Navigation.AdmissionsTab">
    <failure type="org.testng.SkipException" message="Charlie Login Credentials window is appearing DefectID: - ">
      <![CDATA[org.testng.SkipException: Charlie Login Credentials window is appearing DefectID: - 
	at orion3_Listeners.DisableKnownTestFailures.beforeInvocation(DisableKnownTestFailures.java:47)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker$InvokeBeforeInvocationWithoutContextStrategy.callMethod(InvokedMethodListenerInvoker.java:84)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker.invokeListener(InvokedMethodListenerInvoker.java:62)
	at org.testng.internal.Invoker.runInvokedMethodListeners(Invoker.java:619)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:687)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- verifyReassignLeadsPage -->
  <testcase name="verifyEnrollmentExceptionsPage" time="12.869" classname="orion3_Navigation.AdmissionsTab"/>
</testsuite> <!-- orion3_Navigation.AdmissionsTab -->
